spec/rubocop/cop/style/empty_line_after_magic_comment_spec.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+describe RuboCop::Cop::Style::EmptyLineAfterMagicComment do
+  let(:config) { RuboCop::Config.new }
+  subject(:cop) { described_class.new(config) }
+
+  it 'registers an offense for code that immediately follows comment' do
+    inspect_source(cop, ['# frozen_string_literal: true',
+                         'class Foo; end'])
+    expect(cop.offenses.size).to eq(1)
+    expect(cop.messages).to eq(['Add a space after magic comments.'])
+  end
+
+  it 'accepts code that separates the comment from the code with a newline' do
+    inspect_source(cop, ["# frozen_string_literal: true\n",
```

Comment:
What are your thoughts on writing this as

```ruby
inspect_source(cop, ["# frozen_string_literal: true",
                     '',
                     'class Foo; end'])
```

In my mind, it is a little clearer that there is an empty line between the comment and the class.